src/com/xilinx/rapidwright/util/BlackboxPopulator.java

```diff
@@ -152,7 +165,7 @@ public static boolean isClockNet(Net net) {
      * @param top      The design with black boxes to fill
      * @param cellName A black box name
      */
-    public static void combinePIPonClockNets(Design top, String cellName) {
+    private static void combinePIPonClockNets(Design top, String cellName) {
```

Comment:
Can we rename this to `combinePIPsOnClockNets()`?

Reply:
changed